docs/docs/logging.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Logging
+---
+
+# Accessing Log Information
+
+Jenkins utilizes java.util.logging to generate logs. This system sends each log above the logging level INFO to stdout, which is captured and stored by Jenkins and can be viewed in the web GUI. By default, Jenkins contains one log recorder that displays all logs of level INFO or above. 
```

Comment:
We've been using `UI` instead of `GUI`. Would you mind changing instances of `GUI` to `UI` to be consistent?